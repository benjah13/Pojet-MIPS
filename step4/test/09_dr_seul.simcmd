#test la comande dr ; fonctionnement correct
dr
