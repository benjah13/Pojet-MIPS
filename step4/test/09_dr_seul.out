$zero	$0	0	0x0
$at	$1	0	0x0
$v0	$2	0	0x0
$v1	$3	0	0x0
$a0	$4	0	0x0
$a1	$5	0	0x0
$a2	$6	0	0x0
$a3	$7	0	0x0
$t0	$8	0	0x0
$t1	$9	0	0x0
$t2	$10	0	0x0
$t3	$11	0	0x0
$t4	$12	0	0x0
$t5	$13	0	0x0
$t6	$14	0	0x0
$t7	$15	0	0x0
$s0	$16	0	0x0
$s1	$17	0	0x0
$s2	$18	0	0x0
$s3	$19	0	0x0
$s4	$20	0	0x0
$s5	$21	0	0x0
$s6	$22	0	0x0
$s7	$23	0	0x0
$t8	$24	0	0x0
$t9	$25	0	0x0
$k0	$26	0	0x0
$k1	$27	0	0x0
$gp	$28	0	0x0
$sp	$29	0	0x0
$fp	$30	0	0x0
$ra	$31	0	0x0
$pc	$32	0	0x0
$hi	$33	0	0x0
$lo	$34	0	0x0
$sr	$35	0	0x0
